File: EX1/EventMasterSoft/EventMasterSoftDA/DAO/EventoDAO.cs
using EventMasterSoftModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EventMasterSoftDA.DAO
{
    public interface EventoDAO
    {
        int insertar(Evento evento);

        BindingList<Evento> listarPorNombre(string nombre);

        Evento obtenerPorId(int idEvento);
    }
}

File: LAB5/Backend/EventMasterSoftDA/src/pe/edu/pucp/eventmastersoft/dao/EventoDAO.java
package pe.edu.pucp.eventmastersoft.dao;

import java.util.ArrayList;
import pe.edu.pucp.eventmastersoft.model.Evento;

public interface EventoDAO {
    int insertar(Evento evento);
    ArrayList<Evento> listarPorNombre(String nombre);
    Evento obtenerPorId(int idEvento);
}